### ISSUE TITLE
Can't add user to a specific repo or team

### ISSUE THREAD

**Issue by thibaultmeyer** [go-gitea/gitea#361]
When we try to assign user to a specific project or a team, we get the error `user does not exist.`.

![image](https://cloud.githubusercontent.com/assets/1005086/20963237/e9e676fc-bc6c-11e6-8314-b54c25e98e5a.png)

**Comment by Bwko** [comment]
This is caused by commit 55bd3edb90285fe5983229defcaec7b49f94b980
It is caused by:

```
 -	UserName  string `json:"username"`
 +	UserName  string `json:"login"`
```

**Comment by lunny** [comment]
This have been resolved by #362
